### ISSUE TITLE
ModuleNotFoundError: No module named 'expecttest'

### ISSUE THREAD

**Issue by abidwael** [pytorch/pytorch#100002]
### 🐛 Describe the bug

Hi PyTorch team, we're installing the nightly build but are running into `ModuleNotFoundError: No module named 'expecttest'` when we use `torch.compile`.
It looks like `expecttest` is imported in `torch/testing/_internal/common_utils.py:58`.

### Versions

`pip install --pre torch --index-url https://download.pytorch.org/whl/nightly/cpu`

**Comment by wconstab** [comment]
yes, it was actually a forward-fix not a revert, but it landed yesterday https://github.com/pytorch/pytorch/pull/99997 - lmk if anything is still broken
